action-scripts/brownie/scripts/report_gauges.py

```diff
@@ -124,6 +124,8 @@ def _parse_added_transaction(transaction: dict, **kwargs) -> Optional[dict]:
     pool_name, pool_symbol, pool_id, pool_address, a_factor, fee, style = _extract_pool(
         chain, gauge, gauge_selectors
     )
+    bip_number = transaction.get("bip_number")
```

Comment:
Set some default values for better output: 
```suggestion
    bip_number = transaction.get("bip_number", "N/A")
```